packages/spatial/src/input/functions/ClientInputHeuristics.test.tsx

```diff
@@ -301,13 +137,10 @@ describe('ClientInputHeuristics', () => {
 
         const rayOrigin = new Vector3()
         const rayDirection = new Vector3(2, 2, 2)
-        const ray = new Ray(rayOrigin, rayDirection)
-        const hitTarget = new Vector3()
         const data = new Set<IntersectionData>()
-        assert.equal(data.size, 0)
 
-        ClientInputHeuristics.findBBoxes(data, ray, hitTarget)
-        assertVec.approxEq(boxMin, hitTarget, 3)
+        boundingBoxHeuristic(data, rayOrigin, rayDirection)
+        assertFloat.approxEq(1, Array.from(data)[0].distance, 3)
```

Comment:
Probably meant:
```ts
assertFloat.approxEq(1, Array.from(data)[0].distance, 0.001)
```
but `0.001` is the default epsilon for float approxEq, iirc. So it could be omitted

Reply:
Thanks, that was actually hiding a failing test!